### PR TITLE
add second version of the katello testing slides

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: _layouts/media.html [hunk 1]
```diff
@@ -43,7 +43,7 @@ <h1>Other presentations</h1>
 					<li><a href="http://ares.github.io/presentations/kafo/presentation.html#slide1">Kafo - the framework for creating shiny installers based on Puppet</a> (Marek Hul&aacute;n)</li>
 					<li><a href="https://docs.google.com/presentation/d/1nDUmWj_L6Fl6JneHVWq6s8C3UzWD32Rs_scPEqHZHuk/edit?usp=sharing">Build your own image: Foreman discovery plugin</a> (Lukas Zapletal)</li>
 					<li><a href="/static/presentations/evgeni_golov-mass-migration_of_5000_servers_to_foreman_katello_with_bootstrap.pdf">Mass-Migration of 5000 Servers to Foreman/Katello with bootstrap.py</a> (Evgeni Golov)</li>
-					<li><a href="/static/presentations/evgeni_golov-katello_and_ansible_for_automated_testing_and_releasing_of_packages.pdf">Katello and Ansible for automated testing and releasing of packages</a> (Evgeni Golov)</li>
+          <li><a href="/static/presentations/evgeni_golov-katello_and_ansible_for_automated_testing_and_releasing_of_packages.pdf">Katello and Ansible for automated testing and releasing of packages</a> (Evgeni Golov) (<a href="/static/presentations/Evgeni_Golov-Katello_and_Ansible_for_automated_testing_and_releasing_of_packages.pdf">second version of the slides)</li>
```
Comment (ekohl) [inline _layouts/media.html]: Odd indenting

**Review by ekohl — CHANGES_REQUESTED** [review]
There's a merge conflict now

File: _layouts/media.html (hunk 1 above)
Comment (evgeni) [reply _layouts/media.html]: my vim hates tabs :)

**Comment by evgeni** [comment]
updated!

**Review by ekohl — APPROVED** [review]

**Pull request merged**